### ISSUE TITLE
Warning: Using ranger as a file launcher is deprecated

### ISSUE THREAD

**Issue by GloverDonovan** [francoiscabrol/ranger.vim#37]
With the default command `:Ranger`, instead of Ranger opening, I get the following message instead:

```
Warning: Using ranger as a file launcher is deprecated.
Please use the standalone file launcher 'rifle' instead.
```

Interestingly enough, other commands like `:RangerNewTab` work just fine.

Ranger version: ranger-stable 1.8.1

**Comment by francoiscabrol** [comment]
Ok this is weird... I use the same ranger version 1.8.1 and don't know this issue. It's annoying if it breaks the main feature, i will try to reproduce it.

**Comment by francoiscabrol** [comment]
this should be fixed now by the pull request #38

**Comment by GloverDonovan** [comment]
Just updated and everything is working now. 👍
